File: frontend/src/app/model/patient.ts
export interface IPatientDTO {
  id?: string;
  identification?: string;
  firstName?: string;
  lastName?: string;
  email?: string;
}

export class PatientDTO implements IPatientDTO {
  constructor(
    public id?: string,
    public identification?: string | undefined,
    public firstName?: string | undefined,
    public lastName?: string | undefined,
    public email?: string | undefined
  ) {}
}

File: backend/src/main/java/com/alejo0153/patientApi/model/Patient.java
package com.alejo0153.patientApi.model;

import javax.validation.constraints.NotBlank;

public class Patient {

	@NotBlank
	private String id;
	@NotBlank
	private String identification;
	@NotBlank
	private String firstName;
	@NotBlank
	private String lastName;
	@NotBlank
	private String email;
	
	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getIdentification() {
		return identification;
	}

	public void setIdentification(String identification) {
		this.identification = identification;
	}

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

}
